substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JavaLangSubstitutions.java

```diff
@@ -722,6 +745,348 @@ static Package getPackage(Class<?> c) {
     }
 }
 
+@TargetClass(className = "java.lang.StackWalker", onlyWith = JDK9OrLater.class)
```

Comment:
All JDK 9 specific substitutions must be in the `com.oracle.svm.core.jdk9` project. Then you can also use, e.g., `StackWalker.StackFrame` directly without a target class - the substitutions are then compiled with JDK 9 (or actually 11)

Reply:
Will do.